.mergify.yml

```diff
@@ -0,0 +1,25 @@
+queue_rules:
+  - name: default
+    queue_conditions:
+      - "#approved-reviews-by>1"
+      - base=main
+      - label=A:automerge
+    merge_conditions:
+      - "#approved-reviews-by>1"
+    commit_message_template: |
+      {{ title }} (#{{ number }})
+      {{ body }}
```

Comment:
Do we need body? As far as i know body is PR description, and sometimes someone would put quite big description + images...